pkg/db/migrations/2_checks.sql

```diff
@@ -6,6 +6,7 @@ CREATE TABLE canaries (
 	namespace text NOT NULL,
 	labels jsonb NULL,
 	spec jsonb NOT NULL,
```

Comment:
We need this to be persisted in the DB because we need this information in our SyncCanaryJob

So when we fetch the canaries from DB and convert it to V1.Canary we have the status.Checks present

Reply:
But we look that up against the checks table ?

Reply:
No we fetch all the canaries for the sync up and then use the runChecks against that whole canary